test/Response/SapiStreamEmitterTest.php

```diff
@@ -57,76 +61,322 @@ public function testDoesNotInjectContentLengthHeaderIfStreamSizeIsUnknown()
         }
     }
 
-    public function emitBodyProvider()
+    private function expandTestData()
```

Comment:
I get it. I'll use duplication.
I thought the same when I finished writing this method.
I tried a version without array_walk (using for, next, current), it got worse.